bot/seasons/evergreen/wikipedia.py

```diff
@@ -0,0 +1,52 @@
+import logging
+from typing import Optional
+
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+SEARCH_API = "https://en.wikipedia.org/w/api.php?action=query&list=search&srsearch={search_term}&format=json"
+WIKIPEDIA_URL = "https://en.wikipedia.org/wiki/{title}"
+
+
+class WikipediaCog(commands.Cog):
+    """Get info from wikipedia."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+        self.http_session = bot.http_session
+
+    async def search_wikipedia(self, search_term: str) -> Optional[str]:
+        """Search wikipedia and return the first page found."""
+        async with self.http_session.get(SEARCH_API.format(search_term=search_term)) as response:
+            data = await response.json()
+
+        search_result = data["query"]["search"]
+        if len(search_result) == 0:
+            return None
+
+        page = search_result[0]
+
+        # we dont like "may refere to" pages.
+        if "may refer to" in page["snippet"]:
+            log.info(f"hit a 'may refer to' page, taking result 2 (index 1)")
+            page = search_result[1]
```

Comment:
I believe this can use more logic to check the results we got, otherwise we can have this

![image](https://user-images.githubusercontent.com/19921765/78027443-92450f00-7387-11ea-87fb-4fab41cf696a.png)

In case we got too many like when we hit a `may refer to` as the first page, maybe wikipedia does not have it and so we can list the results instead?